semperfiwebdesign/all-in-one-seo-pack
# Fix SEO for search results page

Issue https://github.com/semperfiwebdesign/aioseop-pro/issues/468

## Proposed changes

Fixes an issue where the Pro plugin did not do SEO for the search results page. 

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist


## Testing instructions

First, reproduce the issue using these steps (with a Pro release) - https://github.com/semperfiwebdesign/aioseop-pro/issues/468#issuecomment-497084273

Then install the PR (after building Pro with it) and make sure it fixes the issue.
